pages/highlights/models.py

```diff
@@ -0,0 +1,228 @@
+    
+    # Content fields
+    summary = models.TextField(
+        help_text="Brief summary of the highlight for display in cards"
+    )
+    content = models.TextField(
+        help_text="Full content in markdown format displayed on detail page"
+    )
+    announcement = models.TextField(
+        blank=True,
+        help_text="Optional announcement message displayed at the top of the highlight"
+    )
+    tags = models.TextField(
+        blank=True,
+        help_text="Comma-separated tags for related content matching and search"
+    )
+    author = models.CharField(
+        max_length=255,
+        blank=True,
+        help_text="Author name or names for the highlight (optional)"
+    )
+    
+    # Media fields
+    featured_image = models.ImageField(
+        upload_to="highlights/images/",
+        help_text="Featured image for the highlight"
+    )
+    figure_caption = models.TextField(
+        blank=True,
+        help_text="Caption for the featured image (optional)"
+    )
+    
+    # Relationships
+    topics = models.ManyToManyField(
+        "topics.Topic",
+        blank=True,
+        help_text="Research topics associated with this highlight (optional)"
+    )
+    
+    # Status field
+    is_active = models.BooleanField(
+        default=True,
+        help_text="Whether this highlight is active and visible"
+    )
+    
+    # Timestamps
+    created_at = models.DateTimeField(
+        default=timezone.now,
+        help_text="Creation date (defaults to current date if not provided)"
+    )
+    updated_at = models.DateTimeField(auto_now=True)
+    
+    class Meta:
+        ordering = ["-created_at"]
+        verbose_name = "Data Highlight"
+        verbose_name_plural = "Data Highlights"
+    
+    def __str__(self):
+        """Return the highlight title for string representation."""
+        return self.title
+    
+    def save(self, *args, **kwargs):
+        """Save the highlight, auto-generating slug if not provided."""
+        if not self.slug:
+            self.slug = slugify(self.title)
+        super().save(*args, **kwargs)
+    
+    @property
+    def rendered_content(self):
+        """Return content rendered as HTML from markdown."""
+        return mark_safe(
+            markdown.markdown(self.content, extensions=["extra", "codehilite"])
+        )
+    
+    @property
+    def rendered_announcement(self):
+        """Return announcement rendered as HTML from markdown."""
+        if self.announcement:
+            return mark_safe(
+                markdown.markdown(self.announcement, extensions=["extra", "codehilite"])
+            )
+        return ""
+    
+    @property
+    def tag_list(self):
+        """Return tags as a list of cleaned strings."""
+        if not self.tags:
+            return []
+        return [tag.strip().lower() for tag in self.tags.split(',') if tag.strip()]
+    
+    def get_related_highlights(self, limit: int = 4, threshold: float = 0.1) -> List['DataHighlight']:
```

Comment:
Again a reminder comment, to change the name if we change the model name from _Data Highlights_ to something else.